src/main/java/slimeknights/tconstruct/library/recipe/tinkerstation/building/ToolBuildingRecipeBuilder.java

```diff
@@ -23,8 +24,15 @@ public class ToolBuildingRecipeBuilder extends AbstractRecipeBuilder<ToolBuildin
   private final IModifiable output;
   @Setter
   private int outputSize = 1;
+  @Nullable
+  private ResourceLocation layoutSlot = null;
   private final List<Ingredient> extraRequirements = new ArrayList<>();
 
+  public ToolBuildingRecipeBuilder setLayoutSlot(ResourceLocation layoutSlot) {
```

Comment:
You should be able to just use a `@Setter` on `layoutSlot` for this, note it will name the setter `layoutSlot`, not `setLayoutSlot` but thats fine.

Reply:
I don't understand. Is this not how the builder pattern is done?

Reply:
Lombok lets you generate the builder pattern automatically, look at the other `@Setter` in this class.